Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Conda environments: ignored
Job counts:
	count	jobs
	1	ml_run
	1
Resources before job selection: {'_cores': 1, '_nodes': 9223372036854775807}
Ready jobs (1):
	ml_run
Select jobs to execute...
Selected jobs (1):
	ml_run
Resources after job selection: {'_cores': 0, '_nodes': 9223372036854775806}

[Wed Jun 23 12:01:01 2021]
rule ml_run:
    input: data/gene_data.csv, data/AllLabels.csv
    output: ml/lr.pkl, ml/rf.pkl
    jobid: 0

Terminating processes on user request, this might take some time.
Full Traceback (most recent call last):
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2357, in run_wrapper
    run(
  File "/mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/Snakefile", line 28, in __rule_ml_run
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/shell.py", line 234, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  python scripts/ml_run.py -d data/gene_data.csv -l data/AllLabels.csv lr rf' died with <Signals.SIGINT: 2>.

[Wed Jun 23 12:02:20 2021]
Error in rule ml_run:
    jobid: 0
    output: ml/lr.pkl, ml/rf.pkl
    shell:
        python scripts/ml_run.py -d data/gene_data.csv -l data/AllLabels.csv lr rf
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2357, in run_wrapper
    run(
  File "/mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/Snakefile", line 28, in __rule_ml_run
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/shell.py", line 234, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  python scripts/ml_run.py -d data/gene_data.csv -l data/AllLabels.csv lr rf' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 574, in _callback
    raise ex
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 560, in cached_or_run
    run_func(*args)
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2390, in run_wrapper
    raise RuleException(
snakemake.exceptions.RuleException: CalledProcessError in line 15 of /mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/Snakefile:
Command 'set -euo pipefail;  python scripts/ml_run.py -d data/gene_data.csv -l data/AllLabels.csv lr rf' died with <Signals.SIGINT: 2>.
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2357, in run_wrapper
  File "/mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/Snakefile", line 15, in __rule_ml_run

RuleException:
CalledProcessError in line 15 of /mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/Snakefile:
Command 'set -euo pipefail;  python scripts/ml_run.py -d data/gene_data.csv -l data/AllLabels.csv lr rf' died with <Signals.SIGINT: 2>.
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2357, in run_wrapper
  File "/mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/Snakefile", line 15, in __rule_ml_run
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 574, in _callback
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/concurrent/futures/thread.py", line 57, in run
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 560, in cached_or_run
  File "/home/fernando/mambaforge/envs/snakemake-tutorial-ml/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2390, in run_wrapper
Removing output files of failed job ml_run since they might be corrupted:
ml/lr.pkl, ml/rf.pkl
Complete log: /mnt/c/Users/Fernando/Documents/MRes Biomedical Research/Project 2/Github/BenchmarkDR/workflow/rules/prediction/.snakemake/log/2021-06-23T120100.562912.snakemake.log
unlocking
removing lock
removing lock
removed all locks
